dre-service/src/main/java/io/cdap/re/NamespaceNotFoundException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.re;
+
+/**
+ * Class description here.
+ */
+class NamespaceNotFoundException extends Exception {
+
+  NamespaceNotFoundException(String message) {
```

Comment:
this is not the message, but the namespace that could not be found

Reply:
Done